docs/guide/net.rst

```diff
@@ -183,3 +186,61 @@ Finally, if you know the dataset ID of a dataset you wish to download, you can j
 .. code-block:: python
 
     transfer_complete_datasets("AAAAA")
+
+.. _interpolation-keys:
+
+Path interpolation keys
+-----------------------
+
+When downloading DKIST data with ``ds.files.download()``, the ``path=`` keyword argument can be used to specify the target folder for the download.
+This path can include keys corresponding to metadata entries, and those values are then used to complete the download path.
+For example, to download a dataset into its own folder named with the dataset ID, with separate subfolders for each instrument in the dataset, you could set ``path="~/data/dkist/{Dataset ID}/{Instrument}/"``.
+This would take the values for the ``datasetID`` and ``instrumentName`` metadata keywords in the dataset and insert them into the path string.
+
+Here is a full list of the metadata keywords available for this purpose and their corresponding path interpolation keys:
```

Comment:
Can we auto generate this? There will soon be more keys in here and having to *remember* to update it everytime seems like a bad idea. If we have a Python API for printing it we could call it in a code block in the docs?

Reply:
Yes that's a great call, I'll get on it.